server/src/main/java/org/opensearch/search/approximate/ApproximatePointRangeQueryWeight.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.approximate;
+
+import org.apache.lucene.index.PointValues;
+import org.apache.lucene.search.ConstantScoreWeight;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.util.DocIdSetBuilder;
+
+import java.io.IOException;
+
+/**
+ * Abstract implementation of {@link ConstantScoreWeight} for {@link ApproximatePointRangeQuery}
+ */
+public abstract class ApproximatePointRangeQueryWeight extends ConstantScoreWeight {
```

Comment:
Do we need this class? 

It looks like it only has the one implementation (though I'm just basing that on a Command-F through the page).

Reply:
you're right, not necessary.